lib/eex/test/eex_test.exs

```diff
@@ -175,7 +175,7 @@ defmodule EExTest do
     end
 
     test "when end expression has a modifier" do
-      assert_raise EEx.SyntaxError, ~s[nofile:1: unexpected beginning of EEx tag "<%=" on end of expression "<%= end %>", please remove "=" accordingly], fn ->
+      assert ExUnit.CaptureIO.capture_io :stderr, fn ->
```

Comment:
sorry, I was referring to an already existing _different_ test which that did not check the exception message: https://github.com/wojtekmach/elixir/blob/36b9353aa4a0ebbc1192adf607d3d5e2934b67dd/lib/eex/test/eex_test.exs#L171:L175. Thought it was on purpose or something.

anyhow, will push assertions shortly.